contributing/documentation/standards.rst

```diff
@@ -137,8 +144,13 @@ Files and Directories
 English Language Standards
 --------------------------
 
-* **English Dialect**: use the United States English dialect, commonly called
-  `American English`_.
+Symfony documentation uses the United States English dialect, commonly called
+`American English`_. Disputes over vocabulary will always be resolved using the
+`American English Oxford Dictionary`_ reference. Disputes over neologisms not
+included yet in the dictionary will be resolved by Symfony Documentation maintainers.
```

Comment:
I don't think we need to formalize this section either. The standards should be quick and to the point. Imo, it's clear that the maintainers make the decisions when something is unclear and in any case, I doubt that I have more English knowledge than the PR author introducing words that aren't in the Oxford dictionary.